pkgs/by-name/wi/wiiudownloader/package.nix

```diff
@@ -0,0 +1,68 @@
+# "Building instructions" are at
+# https://github.com/Xpl0itU/WiiUDownloader/blob/main/.github/workflows/linux.yml
+
+{ lib
+, buildGoModule
+, fetchurl
+, fetchFromGitHub
+, pkg-config
+, wrapGAppsHook
+, python3 # for preBuild script
+, gtk3
+}:
+
+let
+
+  # https://github.com/Xpl0itU/WiiUDownloader/blob/main/grabTitles.py
+  gtitles = fetchurl {
+    url = "https://napi.nbg01.v10lator.de/db";
+    hash = "sha256-neJnqAiKloZHlmzuW+h7+EcS+WYRr98dhfv3noCwDmI=";
```

Comment:
Since the output of this URL changes frequently, the hash also does, I already have a hash mismatch:
```console
error: hash mismatch in fixed-output derivation '/nix/store/66b8ysh4v5j385kymp7gw6r8xyfwfav7-db.drv':
         specified: sha256-neJnqAiKloZHlmzuW+h7+EcS+WYRr98dhfv3noCwDmI=
            got:    sha256-I8RSdWJjQUBG4VRBLMOxUn6t3pA2C28dMjBNNhktFQY=
```

Reply:
Ah thanks I didn't realize this. I've changed the package to bundle the gtitles.c file. I'm unsure if this is the best approach though.